src/account/default.ts

```diff
@@ -298,10 +301,36 @@ export class Account extends Provider implements AccountInterface {
         ...callsArray,
       ],
       undefined,
-      transactionsDetail
+      invocationsDetails
     );
   }
 
+  /**
```

Comment:
we can remove doc from here since they are present in the interface

Reply:
466a2aacdac6034b6d5294c1e4ce1208b872acd8